python/konduit/server.py

```diff
@@ -75,8 +91,32 @@ def start(self, sleep=None):
         args = self._process_extra_args(abs_path)
         process = subprocess.Popen(args=args)
         self.process = process
-        if sleep:
-            time.sleep(sleep)
+
+        # Check if the server is up or not.
+        request_timeout = 5
+        tries = int(self.start_timeout/request_timeout + 1)
+
+        started = False
+
+        for i in range(tries):
+            # This url returns status 204 with no content when the server is up.
+            try:
+                print("Checking server integrity. Tries: {} of {}".format(i + 1, tries))
+
+                r = requests.get("http://localhost:{}/healthcheck".format(self.config.serving_config.http_port),
+                                 timeout=request_timeout)
+                if r.status_code == 204:
+                    started = True
+                    break
+            except Exception as ex:
+                logging.debug("{}\nChecking server integrity again...".format(str(ex)))
+
+        if started:
+            print("Server has started successfully.")
```

Comment:
consider using `logging.info` in favor of print

Reply:
how do we change the log level inside the python `logging` module
I added the print statements because it wasn't printing it on the notebooks except for the `error` level

Reply:
oh `logging.getLogger().setLevel(logging.DEBUG)`
I think I'll leave in the print statements because then it will always give out some extra info that I don't want to see with the debug log level

Reply:
I actually want the users to see when the server is up and running